ansible_collections/arista/avd/roles/eos_designs/templates/ip_addressing/avd-v2-spine-p2p-uplinks-ip.j2

```diff
@@ -1,5 +1,5 @@
 {# In AVD v2.x the spine to super-spine links used this special IP addressing scheme #}
-{# This file is still used when using the upgrade role from 2.x to 3.0 #}
+{# This file may still be used by older inventories #}
```

Comment:
Do we want to keep these files as we move to 4.0?

Reply:
I know it is being used in production for older deployments, so yes.